unitybuild-start/action.yml

```
@@ -78,14 +78,22 @@ runs:
         jq -cr '.[].links.start_builds.href' filteredtargets.json \
         | while read -r path; do
           echo "POST $path"
-          http_code=$(curl -s \
-            -w'%{http_code}' \
-            -o tmpoutput.json \
-            -H'Content-Type: application/json' \
-            -H'Authorization: Basic ${{ inputs.api_key }}' \
-            -d"{ $label_json, $commit_json }" \
-            "${{ inputs.base_url }}$path"
-          )
+          attempts=0
+          while [ attempts != 3 ]
+            do
```

1. Могу поспорить ты не проверял код локально в bash. На что/кого надежда?
2. Лучше поместить do на строчку с while, как это **уже** сделано для всех остальных блоков. Принцип подобия кода.
3. Проверка на равенство чревата багами с вечным циклом.

```suggestion
          while [ $attempts -lte 3 ]; do
```